docker.md

```diff
@@ -0,0 +1,46 @@
+# Document title
+
+**Prerequisite knowledge**: web, server, infrastructure, virtualization, bash
```

Comment:
In general, I don't think *web* is required to understand and learn Docker. For example, you can use docker for BI or machine learning purposes.